### PR TITLE
Update Salesforce docsets

### PR DESCRIPTION
**Pull request by ViViDboarder**

### CODE REVIEW

File: docsets/Salesforce_Visualforce/docset.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "Salesforce Visualforce",
-    "version": "208.0",
+    "version": "210.0",
     "archive": "Salesforce_Pages.tgz",
```
Comment (Kapeli) [inline docsets/Salesforce_Visualforce/docset.json]: This archive wasn't updated. You added `Salesforce_Visualforce.tgz` instead.

**Comment by Kapeli** [comment]
What's up with `Salesforce_Combined.tgz`?

**Comment by ViViDboarder** [comment]
Woops! That was accidentally committed. Gone.

File: docsets/Salesforce_Visualforce/docset.json (hunk 1 above)
Comment (ViViDboarder) [reply docsets/Salesforce_Visualforce/docset.json]: Renamed. Thanks.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**